tests/basic_correctness/test_flashinfer.py

```diff
@@ -0,0 +1,49 @@
+"""Compare the short outputs of HF and vLLM when using greedy sampling.
```

Comment:
Can we just use the same way as https://github.com/vllm-project/vllm/blob/4ea1f9678dd93f02424ab3de2149f83a490e6c6f/.buildkite/test-pipeline.yaml#L16 instead? You can do this in selector.py

Reply:
The problem is that flashinfer does not support cudagraph yet (enforce_eager = False will fail the test), do we still want to share the test then?

Reply:
Change test_basic_correctness, skip non-eager mode